### ISSUE TITLE
inspircd-extras rename proposal

### ISSUE THREAD

**Issue by SaberUK** [inspircd/inspircd-contrib#217]
There is currently an ambiguity where the inspircd-extras repository gets mistaken for the extra directory within the InspIRCd source tree which contains non-third party modules that have dependencies on external libraries.

I propose that we fix this by making the following changes:

1. Renaming inspircd-extras to inspircd-contrib.

2. Updating all links in the inspircd-contrib repository to point to the new raw URL.

3. Updating sources.list on the insp20 and insp3 branches to point to the new source list URL.

4. Creating a new archived inspircd-extras repository that contains a sources.list with a single `source` directive in it that points to the URL of the new source list URL.

This should allow all users of older versions to have their module manager keep working whilst also removing the naming ambiguity.

TL;DR:

inspircd-contrib: https://github.com/inspircd/inspircd-extras/compare/master...SaberUK:contrib

inspircd-extras: https://github.com/SaberUK/inspircd-extras

**Comment by SaberUK** [comment]
If there are no constructive objections to this I will enact this on Monday.

**Comment by SaberUK** [comment]
This has been implemented.
